dir.props

```diff
@@ -33,6 +33,8 @@
 
   <PropertyGroup>
     <SharedFrameworkName>Microsoft.NETCore.App</SharedFrameworkName>
+    <NETCoreAppFrameworkMoniker>.NETCoreApp,Version=v2.2</NETCoreAppFrameworkMoniker>
```

Comment:
I like what you've done in master here.  Can we bring it over?

```
    <NETCoreAppFrameworkMoniker>.NETCoreApp,Version=v$(MajorVersion).$(MinorVersion)</NETCoreAppFrameworkMoniker>
    <NETCoreAppFramework>netcoreapp$(MajorVersion).$(MinorVersion)</NETCoreAppFramework>
```

Reply:
Yes I will pull that change over as well.